WordPressCom-Stats-iOS/StatsEphemory.m

```diff
@@ -13,7 +13,7 @@ @implementation EphemoryContainer
 
 @interface StatsEphemory ()
 
-@property (nonatomic, strong) NSCache *cache;
+@property (nonatomic, strong) NSMutableDictionary *cache;
```

Comment:
It has, internally, a NSCache, but you can just go ahead and replace that with NSMutableDictionary.

Before time, we used to persist changes into a plist file... but that killed the app, when dealing with +10k entities.